viz/TrajectoryVisualization.hpp

```diff
@@ -25,6 +25,7 @@ class TrajectoryVisualization: public Vizkit3DPlugin<base::Vector3d>
         TrajectoryVisualization();
         ~TrajectoryVisualization();
         void setColor(const base::Vector3d& color); 
+        void setMaxVelocity(double max_vel);
```

Comment:
Would be best to declare MaxVelocity as a Q_PROPERTY as well, so that it shows in the property browser.